packages/rancher-alerting-drivers/charts/Chart.yaml

```diff
@@ -0,0 +1,45 @@
+name: rancher-alerting-drivers
+home: ""
+sources: []
+version: 1.0.1
+description: The manager for third-party webhook receivers used in the Alertmanger
+keywords:
+- monitoring
+- alertmanger
+- webhook
+maintainers:
+- name: Jiaqi Luo
+  email: ""
+  url: https://github.com/jiaqiluo
+icon: ""
+apiVersion: v2
+condition: ""
+tags: ""
+appversion: 1.16.0
+deprecated: false
+annotations:
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/display-name: Alerting Drivers
+  catalog.cattle.io/experimental: "true"
+  catalog.cattle.io/namespace: cattle-monitoring-system
+  catalog.cattle.io/os: linux
+  catalog.cattle.io/release-name: rancher-alerting-drivers
+kubeversion: ""
+dependencies:
+- name: prom2teams
+  version: ""
+  repository: file://./charts/prom2teams
```

Comment:
CC: @aiyengar2 will this work with the build scripts?

Reply:
I think it should be ok since the `dependencies` section is generated by the script.